templates/admin/process_donations.html

```diff
@@ -14,9 +14,20 @@
 button {
   margin: 3px !important;
 }
+button#refresh.disabled {
+    opacity: 0.5;
```

Comment:
i’m not sure if this just sets a class or actually sets disabled on the button, but using `button#refresh[disabled]` might be cleaner.

alternatively, using `pointer-events: none` to make sure the button doesn’t appear clickable at all could be a nice touch.

Reply:
Yeah IDK why I did this because I 100% should've just added a `disabled` attribute, which I've just committed